tests/test_h2_client.c

```diff
@@ -126,3 +128,31 @@ TEST_CASE(h2_client_connection_preface_sent) {
 
     return s_tester_clean_up();
 }
+
+/* Test that client will automatically send the PING ACK frame back, when the PING frame is received */
```

Comment:
add somewhere /* TODO: test that ping response is sent with higher priority than any other frame */